File: 2696/2696.cs
public class Solution {
    public int MinLength(string s) {
        bool bRem = false;
        while (!bRem)
        {
            int pos1 = s.IndexOf("AB");
            if (pos1 != -1)
            {
                s = s.Remove(pos1,2);
            }
            int pos2 = s.IndexOf("CD");
            if (pos2 != -1)
            {
                s = s.Remove(pos2,2);
            }
            if (pos1 == -1 && pos2 == -1)
            {
                bRem = true;
            }
        }
        return s.Length;
    }
}

File: 2696/2696.py
class Solution(object):
    def minLength(self, s):
        """
        :type s: str
        :rtype: int
        """
        bRem = False
        while bRem == False:
            oldLen = len(s)
            s = s.replace("AB","")
            s = s.replace("CD","")
            if oldLen == len(s):
                bRem = True

        return len(s)
